lib/kafka/offset_manager.rb

```diff
@@ -111,17 +116,51 @@ def seconds_since_last_commit
       Time.now - @last_commit
     end
 
+    def seconds_since_last_recommit
+      Time.now - @last_recommit
```

Comment:
Cleanest thing I could come up with was to remove this method and inline the condition at the calling site with the nil check.